src/background/background.ts

```diff
@@ -1,7 +1,7 @@
 import { IBytesRepository } from "../data/bytes/IBytesRepository";
 import { addBytesTransferred } from "./backgroundHelpers";
 
-chrome.runtime.onMessage.addListener(async (message, sender, sendResponse) => {
+chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
```

Comment:
this function seems okay for now but would be weary of it blowing up - might be useful to split it up and have a separate function call or helper for each message